go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/agentbaker
 
-go 1.22
```

Comment:
Adding the patch version meant I could run go again.

Reply:
I think we are using 1.22.2 with aks-rp